src/components/SharedSocialMedia/SharedSocialMedia.tsx

```diff
@@ -103,16 +104,34 @@ const SharedSocialMedia: React.FC<SharedSocialMediaProps> = ({
             mt: theme.spacing(2.5),
           }}
         >
-          <EmailShareButton url={"youremail@example.com"} title={"Check out this website!"}>
+          <EmailShareButton
+            url={link}
+            title={"Check out this website!"}
+            subject={`Nutrition Facts for ${shareName} `}
+          >
             <Email style={{ width: "44px", height: "44px" }} />
           </EmailShareButton>
-          <FacebookShareButton url={"https://www.facebook.com/"} title={"Check out this website!"}>
+          <FacebookShareButton
+            url={link}
+            quote={`Nutrition Facts for ${shareName}`}
+            hashtag={"nutrition facts"}
+            title={"Check out this website!"}
```

Comment:
`title` is not a valid property for FacebookShareButton

Reply:
fixed